fractal-puppy/Engine
# Feature/looting

To test Enemy Loot
-Chose an enemy and add "EnemyLoot" script to this enemy.
-Add at least one item using this script (You will have a selectable list of GameObjects with ItemPicker script added and a chance of drop for this item)
-Start game, kill this enemy and check items are showed in map.

To test Item Names
-Select a GameObject with an ItemPicker script added ("ItemCollectible1 / 2 / 3")
-Change his rarity
-Start game, check item name is showed in a popup and check item is highlighted when selected with mouse. (Grey/white for "Basic" items, DarkGreen/Green for "Rare", DarkOrange/Orange for "Epic" and DarkPurple/Purple for "Legendary")
